transistor/src/main/webapp/WEB-INF/mybatis-beans.xml

```diff
@@ -33,7 +33,7 @@
 
     <bean id="batchSqlSession" class="org.mybatis.spring.SqlSessionTemplate">
         <constructor-arg index="0" ref="sqlSessionFactory" />
-        <constructor-arg index="1" value="BATCH" />
+        <!--<constructor-arg index="1" value="BATCH" />-->
```

Comment:
Have you pushed this by mistake or do you actually intend to disable batch mode?